status-client-spec.md

```diff
@@ -130,7 +130,7 @@ and at some point we might be forced to change them.
 
 #### Discovery
 
-To implement a Status client you need to discover peers to connect to. We use a
+To implement a Status client you MUST discover peers to connect to. We use a
```

Comment:
Is this accurate? If you have nodes ready as static nodes you don't necessarily need to discover new nodes. SHOULD?

cc @adambabik

Reply:
have a list of static nodes is trivial discovery, imo